### DISCUSSION TITLE
Mage Training Arena - Telekinetic Room

### DISCUSSION THREAD

**Discussion by Cheddabeast** [runelite/runelite#15749] [Q&A] [answered]
After the update implemented by Jagex yesterday, the Mage Training Arena plugin, specifically the Telekinetic Room isn't working. After the first telekinetic grab on the NPC the arrow disappears and doesn't indicate where the player needs to stand next in order to move the NPC through the maze.

**Comment by kamielvf** [comment, marked as answer]
This was fixed in 6af244848326e2283c3a9a966b50d2bafa3c837e
